lib/themes/dosomething/paraneue_dosomething/templates/system/page--full-width.tpl.php

```diff
@@ -12,6 +12,7 @@
 <?php if (!empty($tabs['#primary'])): ?><nav class="admin-tabs"><?php print render($tabs); ?></nav><?php endif; ?>
 <?php print $messages; ?>
 
+<div id="onboarding" class="wrapper"></div>
```

Comment:
Also not absolutely necessary for this PR, but it feels iffy to have a campaign-specific element in the global "page" templates. Maybe we should be specifically overriding the page template for the campaign node?